### ISSUE TITLE
Unable to access "Missing required fields for registration" for pending registration 

### ISSUE THREAD

**Issue by iamszabo** [SAP/gigya-swift-sdk#89]
### Checklist

- [X] The issue can be reproduced in the attached sample application in the repository.
- [X] I have read the relevant [SDK documentation](https://github.com/SAP/gigya-swift-sdk#sap-cdc-gigya-swift-sdk) and have not found a suitable solution or answer.
- [X] I agree to the terms within the [SAP Code of Conduct](https://github.com/SAP/gigya-swift-sdk/blob/main/CODE_OF_CONDUCT.md).

### Description

We have an issue with the Twitter social login. Twitter doesn't return gender or email fields, so we end up with a pending account creation flow.
Previously, we were able to look into what are the missing fields by the following code:
```
if
    case let .gigyaError(gigyaError) = loginApiError.error,
    case let .pendingRegistration(interruption) = loginApiError.interruption,
    let errorDetail = gigyaError.toDictionary()["errorDetails"] as? String,
    errorDetail == "Missing required fields for registration: gender" {
    handleInterruption....
}
```
It seems that the response from the server still contains the `error details` as I can see it in the logs, but the API doesn't expose it in the WebLoginProvider.
```
Log redirect url: gsapi://login_result#errorDetails=Missing+required+fields+for+registration%3a+gender%2c+email&id_token=...&x_regToken=...&error=pending_registration&error_description=206001+-+Account+Pending+Registration
```
I didn't find any information on this from the documentation. Basically my question, what is the currently supported way to access the missing required field in a pending registration flow.
If the API doesn't support it, the only way I can think of is, depending on the social provider, in this case Twitter, we should always ask for email and gender information in case of pending registration.

### Reproduction

I've just mapped social login to Twitter in the example application.

### SDK version

1.7.0

**Comment by levistepanov** [comment]
Is there a Registration Completion screen configured?
https://help.sap.com/docs/SAP_CUSTOMER_DATA_CLOUD/8b8d6fffe113457094a17701f63e3d6a/414d1d2f70b21014bbc5a10ce4041860.html

**Comment by iamszabo** [comment]
@levistepanov No, we don't use screen-sets, we only use custom native screens in our app.
